pkg/sql/sqlbase/structured.proto

```diff
@@ -718,6 +718,18 @@ message TableDescriptor {
   // from the cluster. In nanoseconds since the epoch.
   optional int64 gc_deadline = 29 [(gogoproto.nullable) = false,
            (gogoproto.customname) = "GCDeadline"];
+
+
+  message Replacement {
+    optional uint32 id = 1 [(gogoproto.nullable) = false, (gogoproto.customname) = "ID", (gogoproto.casttype) = "ID"];
+    optional util.hlc.Timestamp time = 2 [(gogoproto.nullable) = false];
```

Comment:
I don't see the Time field being read anywhere. Is it useful? Is it always the same as the table creation time? If we ever do start reusing table ids is the time field enough to determine if we need to backup the previous id?

Reply:
Time is unused by backup/restore code. It's recorded based on @vivekmenezes's suggestion above.